test/JDBC/upgrade/latest/schedule

```diff
@@ -607,3 +607,4 @@ sys_sql_logins
 sys-login-property
 sys-fn-varbintohexsubstring
 BABEL_OBJECT_RESOLUTION_IN_ROUTINES
+BABEL-5608
```

Comment:
BABEL-5608 is only schedule in the latest version upgrade test ? Why ?

Reply:
Added in other upgrade paths as well